backend/mora/user_settings.json

```diff
@@ -1,4 +1,12 @@
-{'orgunit':
-   {'show_family_line': True,
-    'show_bvn': True}
+{
```

Comment:
This file is described in the docs, and we have some sensible defaults, so I don't think we should include it.